vitest.shims.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="@vitest/browser/providers/playwright" />
```

Comment:
not sure why this is here. also not sure why it is commented out. likely has to do with vitest install.

Reply:
afaik we need it to run vitest browser mode tests